shared/common-adapters/index.js

```diff
@@ -21,6 +21,7 @@ export {default as BoxGrow} from './box-grow'
 export {default as Checkbox} from './checkbox'
 export {default as ChoiceList} from './choice-list'
 export {default as ClickableBox} from './clickable-box'
+// TODO: doesn't work: export {default as ConfirmModal} from './confirm-modal/index'
```

Comment:
Note: I spent hours stuck on this and never got it working - any suggestions?

Reply:
Hmm what exactly doesn't work? `yarn flow` works for me here, although I'm getting `Cannot resolve module ./really-leave-team/index.stories.` in `teams/index.stories.js`. That file doesn't exist it seems.

Reply:
idk, storybook just stopped working with no explanation when that was in

Reply:
oh and also, I had forgotten that file, sorry, it's there now

Reply:
Oh I think I know why now. This is because in confirm-modal/index you are importing from `..`, which resolves to `common-adapters/index` and there's an import loop there. I tried importing those components locally in `confirm-modal` and it started working again.

Reply:
thanks Song, I think I got it

Reply:
:white_check_mark: